packages/client/src/components/app/ProgressBar.svelte

```diff
@@ -0,0 +1,33 @@
+<script>
+  import { getContext } from "svelte"
+  import ProgressBar from "../../../../bbui/src/ProgressBar/ProgressBar.svelte"
```

Comment:
It seems like you have not committed this file @poirazis - there should be a file in `bbui`.

Reply:
I didn't make any changes in BBUI, I am just importing the ProgressBar implementation